### PR TITLE
Prepare setup.py for deployment

### PR DESCRIPTION
**Pull request by LD250**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -11,11 +11,18 @@
 with open('CHANGELOG.md') as changelog_file:
     changelog = changelog_file.read()
 
-requirements = []
-
-setup_requirements = ['pytest-runner', ]
-
-test_requirements = ['pytest', ]
```
Comment (LKaemmerling) [inline setup.py]: Why did you drop pytest here?

File: requirements/test.txt [hunk 2]
```diff
@@ -1,6 +1,5 @@
 -r base.txt
```
Comment (LKaemmerling) [inline requirements/test.txt]: Why did you rename the file? Aren't this the dev requirements? I think dev requirements should include the test requirements.

File: setup.py (hunk 1 above)
Comment (LD250) [reply setup.py]: We need these requirements only if we run tests like `python setup py test`. And we do not do this.

File: requirements/test.txt (hunk 2 above)
Comment (LD250) [reply requirements/test.txt]: Projects requirements are in setup.py itself now. To install them run `pip install -e .`.
And `requirements/test.txt` are only used to run tests. that why I renamed it.

**Comment by LKaemmerling** [comment]
LGTM
